develocity-gradle.yml

```diff
@@ -200,65 +224,133 @@ spec:
+                          buildScanExtension.value CI_AUTO_INJECTION_CUSTOM_VALUE_NAME, ciAutoInjectionCustomValueValue
+                          if (isAtLeast(develocityPluginVersion, '2.1') && atLeastGradle5) {
+                              if (isAtLeast(develocityPluginVersion, '3.17')) {
+                                  buildScanExtension.capture.fileFingerprints = develocityCaptureFileFingerprints
+                              } else if (isAtLeast(develocityPluginVersion, '3.7')) {
+                                  buildScanExtension.capture.taskInputFiles = develocityCaptureFileFingerprints
                               } else {
-                                  buildScan.captureTaskInputFiles = geCaptureTaskInputFiles
+                                  buildScanExtension.captureTaskInputFiles = develocityCaptureFileFingerprints
                               }
                           }
                       }
 
-                      if (geUrl && geEnforceUrl) {
-                          pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID) {
-                              afterEvaluate {
-                                  logger.lifecycle("Enforcing Gradle Enterprise: $geUrl, allowUntrustedServer: $geAllowUntrustedServer")
-                                  buildScan.server = geUrl
-                                  buildScan.allowUntrustedServer = geAllowUntrustedServer
+                      if (develocityUrl && develocityEnforceUrl) {
+                          logger.lifecycle("Enforcing Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer, captureFileFingerprints: $develocityCaptureFileFingerprints")
+                      }
+  
+                      pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID) {
+                          afterEvaluate {
+                              if (develocityUrl && develocityEnforceUrl) {
+                                  buildScan.server = develocityUrl
+                                  buildScan.allowUntrustedServer = develocityAllowUntrustedServer
+                              }
+                          }
+  
+                          if (buildScanTermsOfUseUrl && buildScanTermsOfUseAgree) {
+                              buildScan.termsOfServiceUrl = buildScanTermsOfUseUrl
+                              buildScan.termsOfServiceAgree = buildScanTermsOfUseAgree
+                          }
+  
+                          if (isAtLeast(develocityPluginVersion, '2.1') && atLeastGradle5) {
+                              if (isAtLeast(develocityPluginVersion, '3.7')) {
+                                  buildScan.capture.taskInputFiles = develocityCaptureFileFingerprints
+                              } else {
+                                  buildScan.captureTaskInputFiles = develocityCaptureFileFingerprints
                               }
```

Comment:
This is not needed as we should set this only if the Dv plugin is not already applied (see https://github.com/gradle/develocity-gitlab-templates/pull/24 description)

Reply:
Done in https://github.com/gradle/develocity-gitlab-templates/pull/25/commits/41f7a89ea28c0773d451d46ccce02701d6bf074b